test/src/test/java/org/corfudb/runtime/clients/ManagementClientTest.java

```diff
@@ -80,13 +77,6 @@ public void handleBootstrap()
                 .isInstanceOf(ExecutionException.class);
     }
 
-    @Test
-    public void addNodeWorkflowRPCTest() throws Exception {
```

Comment:
Is this test not valid anymore or?

Reply:
It is, I just moved the logic to the other test.